yarn.lock

```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@^7.0.0-beta.35":
   version "7.0.0-beta.42"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0-beta.42.tgz#a9c83233fa7cd06b39dc77adbb908616ff4f1962"
-  integrity sha512-L8i94FLSyaLQpRfDo/qqSm8Ndb44zMtXParXo0MebJICG1zoCCL4+GkzUOlB4BNTRSXXQdb3feam/qw7bKPipQ==
```

Comment:
Can you revert the removal of this field? I think the easiest way is to revert this file to its previous state, then use the repository build to generate the lockfile (`yarn build && ./bin/yarn`).

Reply:
Removal of these lines are not intentional. In fact these lines were not supposed to be there, as they were accidentally generated by unreleased yarn 1.8 in #5042. As yarn 1.8's not released, I can't generate a lockfile with these lines. See discussion here https://github.com/yarnpkg/yarn/pull/5042#issuecomment-393123599. @rally25rs 
 then pushed a commit 41040f1d21bb792d4cbb344a9d2e9f4801e1af7a to revert the addition of these lines, and I merged the commit (2nd commit in this PR). So I believe this 2 commits together is a clean PR.